.github/workflows/CI.yml

```diff
@@ -41,15 +41,19 @@ jobs:
           version: ${{ matrix.version }}
           arch: ${{ matrix.arch }}
       - uses: julia-actions/cache@v1
-        with:
-          cache-name: CI - Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }}
-          cache-compiled: true
+      - run: sudo apt-get update && sudo apt-get install -y xorg-dev mesa-utils xvfb libgl1 freeglut3-dev libxrandr-dev libxinerama-dev libxcursor-dev libxi-dev libxext-dev
```

Comment:
Why did you delete this?

Reply:
I think I wanted to simply add the new one, I can add it again, sorry.

Reply:
Do we really need all those dependencies?

Reply:
I don't think so, sorry. This is for the GLMakie backend. I will remove this.